### ISSUE TITLE
plantumlconfig is looking in the wrong dir

### ISSUE THREAD

**Issue by philsturgeon** [asciidoctor/asciidoctor-epub3#190]
HTML and PDF mode are generating my diagrams ok (thanks to asciidoctor/asciidoctor-pdf#1033) but epub has an interesting error. With a book that looks like this:

```
= Foo
:doctype: book
:source-highlighter: coderay
:toc:
:plantumlconfig: ./plantuml.cfg

:imagesdir: 1-theory

include::1-theory/part.adoc[leveloffset=+1]
```

Then inside that part I include chapters, and inside there I have some diagrams:

```
[plantuml]
---------------------------------------------------------------------
@startuml

title Accepting Payments: Blocking the GUI and Synchronous

boundary "Web GUI" as GUI
control "API"
boundary "Payment Gateway" as PG

GUI -> API: POST /payments
API -> PG: POST /submitPaymentAttempt
PG -> PG: May take 30-60s to confirm
API <-- PG: HTTP 200 OK
GUI <-- API: HTTP 200 OK

@enduml
---------------------------------------------------------------------
```

When I generate epub I get this:

```
$ asciidoctor-epub3 -r asciidoctor-diagram -D generated src/book.adoc
asciidoctor-diagram: ERROR: Failed to generate image: PlantUML image generation failed: /Users/phil/src/book-build-apis-2/src/1-theory/plantuml.cfg (No such file or directory)
asciidoctor-diagram: ERROR: Failed to generate image: PlantUML image generation failed: /Users/phil/src/book-build-apis-2/src/1-theory/plantuml.cfg (No such file or directory)
asciidoctor-diagram: ERROR: Failed to generate image: PlantUML image generation failed: /Users/phil/src/book-build-apis-2/src/1-theory/plantuml.cfg (No such file or directory)
asciidoctor-diagram: ERROR: Failed to generate image: PlantUML image generation failed: /Users/phil/src/book-build-apis-2/src/1-theory/plantuml.cfg (No such file or directory)
```

It seems to be looking in the the imagedir configured for that section, instead of... well not looking in there. No reason for a config file to look in an image directory right?

**Comment by mojavelinux** [comment]
I believe this is a duplicate of (or at least similar to) #188. It's actually caused by a change in Asciidoctor Diagram which was incompatible with the logic in EPUB3. Thus, the EPUB3 converter will need to be upgraded.

**Comment by mojavelinux** [comment]
Okay, so this is a different issue after all.

However, this looks like an Asciidoctor Diagram issue, not an Asciidoctor EPUB3 issue. What happens when you just convert to HTML?

**Comment by philsturgeon** [comment]
HTML works fine, and PDF works thanks to #1033.

**Comment by mojavelinux** [comment]
Hmm. Very strange that the EPUB3 converter differs from the HTML converter. That's unexpected. I'll have a closer look.

**Comment by slonopotamus** [comment]
When this is fixed, undo https://github.com/asciidoctor/asciidoctor-gradle-plugin/commit/b990581ff50f3077de9bc0feb43da2e44a18682b because it trades one class of problems for different class of problems: chapter files are not copied to intermediate workdir.

asciidoctor-gradle-plugin [claims to copy all resources](https://github.com/asciidoctor/asciidoctor-gradle-plugin/blob/release_3_0_0/base/src/main/groovy/org/asciidoctor/gradle/base/AbstractAsciidoctorBaseTask.groovy#L472) to intermediate workdir, but in fact [only copy `images` dir](https://github.com/asciidoctor/asciidoctor-gradle-plugin/blob/release_3_0_0/base/src/main/groovy/org/asciidoctor/gradle/base/AbstractAsciidoctorBaseTask.groovy#L665).

**Comment by slonopotamus** [comment]
Okay, the problem here is that `base_dir` is not passed to chapter docs. This is going to change in #282.

**Comment by slonopotamus** [comment]
I believe this issue will be automagically fixed when we implement #47.
